packages/sui-studio-utils/src/domain-builder/index.js

```diff
@@ -61,10 +62,18 @@ export default class DomainBuilder {
                 return self._useCases[useCase].success !== undefined
                   ? Promise.resolve(
                       typeof successResponse === 'function'
-                        ? successResponse(params)
+                        ? inlineError
```

Comment:
I see here tons of conditions but not a single test, it would be nice to have a test to, at least, check if it keeps the backward compatibility with the clients using the old version of the API